app/helpers/application_helper.rb

```diff
@@ -5,6 +5,13 @@ def default_meta_tags
     I18n.t(:meta).merge(separator: '-', author: '/humans.txt')
   end
 
+  def link_to_locale(locale, &block)
+    options = (I18n.default_locale == locale) ? nil : { locale: locale }
+    html_options = { rel: 'alternate', hreflang: locale }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.